src/ray/protobuf/common.proto

```diff
@@ -116,6 +116,8 @@ message TaskArg {
   bytes data = 2;
   // Metadata for pass-by-value arguments.
   bytes metadata = 3;
+  // ObjectIDs that were inlined in the data field.
+  repeated bytes inlined_ids = 4;
```

Comment:
To clarify, does this include the object id itself? Or just nested ids?

Reply:
Ah I guess it is already present in object_ids. Consider renaming this field to nested_inlined_ids for clarity.